common/openapi.yml

```diff
@@ -408,11 +453,11 @@ components:
           items:
             $ref: '#/components/schemas/JobReport'
 
-    JobId:
```

Comment:
Ah, no, its reference got removed in my previous PR (once we properly return a JobControl with `id` field as the response to creating a new job, rather than just the jobId) and I failed to remove this. you can see that here: https://github.com/DataBiosphere/terra-scientific-pipelines-service/pull/48/files#diff-e862a1565882d38762f5b513f134435a5f220d06a8374341f5200de8a61ff131L264 (old row 264 in openapi.yml)